docs/application/native/guides/ui/tizen-ws-shell/tzsh-softkey.md

```diff
@@ -0,0 +1,207 @@
+# TZSH-Softkey
+
+TZSH-Softkey is a library to control a Softkey service window that shows the software buttons such as back button and home button. You can use TZSH-Softkey to control the Softkey service window's visible state, expandable state and opacity state.
+Each of states are have following properties:
+
+- **Visible State**: You can show or hide the Softkey service window. If you set it as show, the Softkey service window appears. Else if you set it as hide, the Softkey service window disappears.
+
+- **Expandable State**: You can control expandable state of the Softkey service window.
+If you set the expand state to on, then expandable button of the Softkey service window appears as follows:
+![Expand On Transparent](./media/tzsh_softkey_expand_on_transparent.png)
+if you set the expand state to off, then expandable button disppears as follows:
+![Expand Off Transparent](./media/tzsh_softkey_expand_off_transparent.png)
+
+- **Opacity State**: You can control the opacity state of the Softkey service window.
+If you set the opacity state to opaque, the background of the Softkey service window changes to 24-bit color opaque window as follows:
+![Expand Off Opaque](./media/tzsh_softkey_expand_off_opaque.png)
+if you set the opacity state to transparent, the background changes to 32-bit colored transparent window as follows:
+![Expand Off Transparent](./media/tzsh_softkey_expand_off_transparent.png)
+
+## Prerequisites
+
+To use the functions and structures of the TZSH-Softkey API (in mobile and wearable applications), include the `<tzsh_softkey.h>` header file in your application:
+
+```
+#include <tzsh_softkey.h>
+```
+
+## Create TZSH-Softkey Handler
+
+After you have created the main window of your application, call `tzsh_create and tzsh_softkey_create` with a native window ID to create the `tzsh_softkey_h` structure (in mobile and wearable applications).
+	
+```
+#include <Elementary.h>
+
+static void
+init(const char *name)
+{
+   Evas_Object *main_win;
```

Comment:
Because of the sample codes following EFL coding conventions, use 3 spaces as indentation.
(For more information, please refer: coding-conventions)
If it needs to following other coding rules, i'll modify them.
please, verify this.

Reply:
I changed to use 4 spaces as indentation of whole sample codes.
Thanks